### ISSUE TITLE
Play Test Report

### ISSUE THREAD

**Issue by davidcmoulton** [GeorgiaHCA/OpenScienceGame#9]
## Rules / Cards Used

Based on https://github.com/GeorgiaHCA/OpenScienceGame/pull/7 with the following rules modifications:

- **Roles** allocated randomly to get over the uncertainty of committing to the unknown with a new game.

**Tool shop rule modification**
At the end of each round any remaining cards in the **Tool shop** were discarded and the **Tool shop** replenished with the top 3 cards from the **Tool** deck.

**Motivation for tool shop rule modification**
It wasn't clear how the **Tool box** is supposed to restock to 3 cards. As the **Tool shop** got stuck very early anyway, when the 3 cards were either duplicates or replicated the ability of one of the **Roles** in play, we decided to try the discard / replenish mechanic in order to expose the **Tools** to the players.

**First player rotation**
At the beginning of the game the first player is given the **First Player Token**, allowing them to take the first turn of the round. The last action of a round is to pass the **First Player Token** to the next **Player** clockwise.

**Motivation for First player rotation**
When playing with the **Tool box** discard / replenish mechanic, First player rotation is required to prevent the same player always having the first shot at the **Tool box**.

## Players

2 players: 
- Player A: ex developmental biologist, now a science publishing professional
- Player B: ex biochemist, now a science publishing professional and software developer (me)

## Game Report
- Player A: PhD Student
- Player B: Post Doc

The **Tool box** clogged up very quickly, and we house ruled our way out of that (see Rules section above). This did lead to some good moments of tension when considering a decision: if you don't take a **Tool** card, it won't be available next turn, but its duplicate might be, but only might be, but how much do you want that tool vs to do some other thing...

All but one **Insight** came out very late in the game, but we managed to pull it off and win in the very last turn.

Player A achieved their personal objective.

Player B did not achieve their personal objective. But only because they kept getting distracted taking notes ;-)

The way the restrictive **Events** and the helpful **Tools** acted in tension with each other worked well. At one point there it looked like the game was deadlocked, but the **Teach** action saved the day.

The compulsory take-a-card mechanic adds some tension as the game clock. This worked well.

## Overall Impressions
- Player A enjoyed the game, and is looking forward to playing it again when the cards hold more themed content.

- General feeling it would have been more fun with more players.

## Detailed Feedback
- Although it worked mechanistically (just) as a 2 player game, it wasn't as fun as the play tests with more players. Three contributing factors were the limited number of roles in play leading to limited variey in playing styles; the sheer number of tools that each player had (even though not all of them were used), and the cognitive load of managing half a deck of dependency trees each making it feel a bit fiddly.

- Little motivation to use the **Teach** action with 2 players and that many tools (it happened once in the whole game). 

- 2 random **Starting Ideas** that excluded D / E / F, and no dedicated **Insights**, worked well. 

- Player A said they thought that having the **Idea** sets colour coded rather than the **Idea** types would be helpful (types might be distinguisehd by icon).

- Player A said they would like to be easily able to distinguish between the **Tools** that help you Reaearch more easily and the **Tools** that let you Disseminate more easily.

### Card clarifications
Some comments and questions regarding cards where the game effects are not totally clear:

- **Tool** "Pre-print server" should make it easier to disseminate using **Open accesss tokens** only, not with **Restricted access tokens** too.
- **Event** "Harware failure" should only affect _researched_ **Data Sets**?
- **Global event** ""Institutional bias" only affects _non-disseminated_ research outputs?
- **Event** "Collaborate at a conference" leaves it up to the player to decide how many **Ideas** they want to swap?

## Suggestions for New Mechanics
- The **Tool box** feels like it needs fixing:
  -  Could consider the discard / replenish mechanic along with **First player rotation**. This had the effect of focusing the attention on the **Tools**, and the game seemed to fall into 2 phases: first the learn-lots-of-tools phase, followed by the do-the-science phase. Should the discard / replenish **Tool Box** mechanic be adopted, the number of cards in the **Tools** deck needs be a multiple of 3.

  - A simpler approach might be to increase the **Tool box** to, say, 5 cards and just replace cards as they are taken.

- Player A suggested decoupling the personal objectives from the **Role** cards and allocating them independently. Would lead to a lot more potential variation (and probably please Professors who do like open science.) Might require a rethink around the characterisation in the **Role** cards and the stories we want them to tell. Might also mean a lot more play testing as many more combinations.

## Suggestions for New Content
Player A suggested a couple of new cards:

New **Event**:
**Title**: preLights
**Flavour text**: Your paper has been featured in preLights
**Game effect**: You may give an **Idea** of your choice to a player of your choice.

New **Event**:
**Title**: Mendeley
**Flavour text**: Someone shares a paper with you in Mendeley
**Game effect**: You may request a specific **Idea** from the player of your choice, who may decide to give it to you.

**Comment by davidcmoulton** [comment]
@barney-walker let me know if you'd like me to raise a PR for the tool box and first player rotation rules into your `updated-tool-mechanic` branch.

**Comment by barney-walker** [comment]
Thanks for play testing! 

In response to your points:

1. Sorry it wasn't clear in the rules, we played that when a Tool is taken it is immediately replaced by a new one from the Deck.
2. We also found that the Tool Box (I like that wording better good suggestion!) got clogged. Your solution could work but it does require the extra token mechanic. My friend suggested just being able to spend an Action to Refresh the Tool Box (which sweeps away the three that are there and replaces them with new ones).
3. Just to clarify did you play the Rule about only having 2 active Tools at a time? Do you just mean it was hard to manage lots of Tools in your Hand?
3. In my Play Test the Players also found it hard to keep track of things when their Hands got big. I think it would be good to test out a Hand Limit mechanic. This would make it easier for Players to keep track of the different actions they can do, incentivise people to Research anything in their hand as soon as possible, do more Teaching and more Exchanging Ideas and would introduce some more strategic decisions about what to discard when they reach hand limit. It needs to be tested to make sure that it doesn't become too easy for Players to accidentally lock up the dependency tree by discarding things. We may need a few Tools / Events which allow people to dig around in the Idea Discard pile. I also like the narrative that you can only hold so many Ideas in your head at once before you Forget them. The Tool could be some sort of note taking app that allows you to revisit old Forgotten Ideas and put them back in your hand for an Action.  
4. The reason I decided to color code the Idea Types rather than the sets is that there are only 4 types whereas there may be any number of variant sets in the future. Even now not all of the Insights are part of complete sets. There is no reason why the dependency tree always has to be strictly linear, it may play better if the dependency tree is more like a network to avoid gridlock.

**Comment by barney-walker** [comment]
I also think we definitely need to figure out how the game scales with number of Players. 

With 2 Players there shouldn't be the same number of Events or Ideas because it makes it too hard to manage the dependency tree and one Player can get hit with ALL the crippling Events making it really hard.

**Comment by davidcmoulton** [comment]
> we played that when a Tool is taken it is immediately replaced by a new one from the Deck

Okay, will try that next time (presuming it still applies then).

>  spend an Action to Refresh the Tool Box (which sweeps away the three that are there and replaces them with new ones)

Interesting idea. Would delay victory slightly but doesn't sound like a significant (game) time sink 🤞 

> Just to clarify did you play the Rule about only having 2 active Tools at a time? Do you just mean it was hard to manage lots of Tools in your Hand?

Ah, no, I missed that rule, we just went for it with allTheToolz. Thinking about it, a 2 tool limit would seem to make the game harder when playing with fewer players, as fewer tools would be out. Have you played with this restriction? Did it slow the game significantly?

If we instituted a hand limit I think we'd need duplicate sets of cards plus the search-the-discard-deck mechanic. Would require some play testing to work out how to fine tune this and not risk too many deadlocks. 👍 to the Forgetting Ideas concept as the hand limit narrative.

**Comment by barney-walker** [comment]
Yeah we played with the 2 Tool limit. Hard to tell if it slowed the game down, we did reach the end of the Deck a turn or two before we would have won but it didn't seem too restrictive and it did make it much easier to manage your stuff (see below)

![DSC_0068](https://user-images.githubusercontent.com/24464416/64738975-bbd04300-d4e8-11e9-9781-23fb11798452.JPG)

**Comment by barney-walker** [comment]
It just gets a bit too hard to calculate the cost of things when you have loads of Tools. They are easy to forget about. 

Unlimited Tools might actually be easier if the Tools were binary, you can't do Actions at all unless you have the Tool (in which case it costs one Action). I'd be interested to Play Test that, it might just make it too easy to reach gridlock though.

**Comment by davidcmoulton** [comment]
Yeah, it turned into a bit of a logistics exercise with all the Tools yesterday, not so much fun.

At the Sprint we did play with some restrictions preventing research  of a certain output type, and that did quickly cause deadlock.
